packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -150,11 +152,17 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
 					await ordererRestWrapper.post<
 						{ id: string; token?: string; session?: ISession } | string
 					>(`/documents/${tenantId}`, {
-						summary: convertSummaryToCreateNewSummary(appSummary),
+						summary: convertSummaryTreeToWholeSummaryTree(
+							undefined,
+							appSummary,
+							"",
+							"",
+						),
 						sequenceNumber: documentAttributes.sequenceNumber,
 						values: quorumValues,
 						enableDiscovery: this.driverPolicies.enableDiscovery,
 						generateToken: this.tokenProvider.documentPostCreateCallback !== undefined,
+						enableWholeSummaryTreeOnFirstSummary: true,
```

Comment:
I'd suggest shorter name.

Reply:
The other one I had in mind was enableAnyBinaryBlobOnFirstSummary but it is not as short as I'd like,